zeek/assets/logs/zeek.yaml

```diff
@@ -134,6 +134,34 @@ facets:
     path: zeek.auth_attempts
     source: log
     type: integer
+  - facetType: list
+    groups:
+      - Zeek
+    name: Client Requested Address
+    path: zeek.request.host
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Zeek
+    name: Client Requested Port
+    path: zeek.request_p
+    source: log
+    type: string
```

Comment:
Looking at all the samples, we cannot find any property in the sample matching this. Are you sure you are using these properties.
And even more, as it seems to be optional property, facets are not really advised in this scenario.

Reply:
The sample is available, can you please have a look and we need this facets since the request_p and request.host need more readable values when displayed in dashboards.